File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {

        int length=nums.length;
        int i=0;
        int j=0;
        while(i<=j && j<length){
            if(nums[j]!=nums[i]){
                int temp=nums[i+1];
                nums[i+1]=nums[j];
                nums[j]=temp;
                i+=1;
                j+=1;
            }else{
                j+=1;
            }
            
            
        }
        return i+1;
        
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i=0
        j=1
        n=len(nums)
        while j<n:
            if nums[j]==nums[i]:
                j=j+1
            else:
                i=i+1
                nums[i]=nums[j]
                j=j+1
        return i+1
        